tests/testthat/test-sankeydiagram.R

```diff
@@ -1234,3 +1234,56 @@ test_that("Label truncation keeps nodes", {
     expect_equal(nrow(sdat100), nrow(sdat10))
     expect_equal(sdat100$nodes$group, sdat10$nodes$group)
 })
+
+dat <-structure(list(`1` = c("Reconomy", "Reconomy", "Reconomy", "Reconomy",
```

Comment:
Is there supposed to be a test with this data?

Reply:
Oops, I didn't think I committed that file. I'm going to put in a Standard R test because we don't have snapshot tests in this repo